torch/csrc/dynamo/eval_frame.c

```diff
@@ -794,8 +812,15 @@ static PyObject* _custom_eval_frame(
 
   // TODO(alband): This is WRONG for python3.11+ we pass in a _PyInterpreterFrame
   // that gets re-interpreted as a PyObject (which it is NOT!)
+  PyObject *plan = get_plan(frame->f_code);
```

Comment:
We used the name 'plan' in absence of a better name, but seeing the code makes it clear that plan is a bad name. How about 'FrameState'?

Reply:
There's nothing plan-y about this, and the fact that you have the plan repped as a dict means we could put literally anything in it.

Reply:
> we could put literally anything in it.

Same with any obj? :P

Anyway, point taken, sg, let's give it a real object